### PR TITLE
docs: new cem for Topic-R components - Page, Slider, RangeSlider, TextArea, Toast

### PR DESCRIPTION
**Pull request by nikoletavnv**
Using custom-elements-manifest for documenting Page, Slider, RangeSlider, TextArea, Toast web components and all related to them child components, enumerators and interfaces. 

Issue: https://github.com/SAP/ui5-webcomponents/issues/7610

### CODE REVIEW

File: packages/main/src/RangeSlider.ts [hunk 1]
```diff
@@ -91,12 +80,12 @@ type AffectedValue = "startValue" | "endValue";
...
  *
  * @constructor
- * @author SAP SE
- * @alias sap.ui.webc.main.RangeSlider
- * @extends sap.ui.webc.main.SliderBase
- * @tagname ui5-range-slider
+ * @extends SliderBase
  * @since 1.0.0-rc.11
  * @public
+ * @csspart progress-container - Used to style the progress container(the horizontal bar which visually represents the range between the minimum and maximum value) of the <code>ui5-range-slider</code>
```
Comment (simonarangelova) [inline packages/main/src/RangeSlider.ts:86]:
```suggestion
 * @csspart progress-container - Used to style the progress container, the horizontal bar that visually represents the range between the minimum and maximum values, of the <code>ui5-range-slider</code>.
```

File: packages/main/src/RangeSlider.ts [hunk 2]
```diff
@@ -91,12 +80,12 @@ type AffectedValue = "startValue" | "endValue";
...
+ * @csspart progress-bar - Used to style the progress bar, which shows the progress of the <code>ui5-range-slider</code>
```
Comment (simonarangelova) [inline packages/main/src/RangeSlider.ts:87]:
```suggestion
 * @csspart progress-bar - Used to style the progress bar, which shows the progress of the <code>ui5-range-slider</code>.
```

File: packages/main/src/RangeSlider.ts [hunk 3]
```diff
@@ -91,12 +80,12 @@ type AffectedValue = "startValue" | "endValue";
...
+ * @csspart handle - Used to style the handles of the <code>ui5-range-slider</code>
```
Comment (simonarangelova) [inline packages/main/src/RangeSlider.ts:88]:
```suggestion
 * @csspart handle - Used to style the handles of the <code>ui5-range-slider</code>.
```

File: packages/main/src/Slider.ts [hunk 4]
```diff
@@ -78,12 +67,12 @@ import {
...
  *
  * @constructor
- * @author SAP SE
- * @alias sap.ui.webc.main.Slider
- * @extends sap.ui.webc.main.SliderBase
- * @tagname ui5-slider
+ * @extends SliderBase
  * @since 1.0.0-rc.11
  * @public
+ * @csspart progress-container - Used to style the progress container(the horizontal bar which visually represents the range between the minimum and maximum value) of the <code>ui5-slider</code>.
```
Comment (simonarangelova) [inline packages/main/src/Slider.ts:73]:
```suggestion
 * @csspart progress-container - Used to style the progress container, the horizontal bar that visually represents the range between the minimum and maximum values, of the <code>ui5-slider</code>.
```

File: packages/main/src/Slider.ts [hunk 5]
```diff
@@ -78,12 +67,12 @@ import {
...
+ * @csspart progress-bar - Used to style the progress bar, which shows the progress of the <code>ui5-slider</code>
+ * @csspart handle - Used to style the handle of the <code>ui5-slider</code>
```
Comment (simonarangelova) [inline packages/main/src/Slider.ts:75]:
```suggestion
 * @csspart handle - Used to style the handle of the <code>ui5-slider</code>.
```

File: packages/main/src/Slider.ts [hunk 6]
```diff
@@ -78,12 +67,12 @@ import {
...
  * @constructor
- * @author SAP SE
- * @alias sap.ui.webc.main.Slider
- * @extends sap.ui.webc.main.SliderBase
- * @tagname ui5-slider
+ * @extends SliderBase
  * @since 1.0.0-rc.11
  * @public
+ * @csspart progress-container - Used to style the progress container(the horizontal bar which visually represents the range between the minimum and maximum value) of the <code>ui5-slider</code>.
+ * @csspart progress-bar - Used to style the progress bar, which shows the progress of the <code>ui5-slider</code>
```
Comment (simonarangelova) [inline packages/main/src/Slider.ts:74]:
```suggestion
 * @csspart progress-bar - Used to style the progress bar, which shows the progress of the <code>ui5-slider</code>.
```

File: packages/main/src/TextArea.ts [hunk 7]
```diff
@@ -66,25 +66,15 @@ type ExceededText = {
...
  *
  * @constructor
- * @author SAP SE
- * @alias sap.ui.webc.main.TextArea
- * @extends sap.ui.webc.base.UI5Element
- * @tagname ui5-textarea
+ * @extends UI5Element
+ * @implements {IFormElement}
  * @public
+ * @csspart textarea - Used to style the native textarea
```
Comment (simonarangelova) [inline packages/main/src/TextArea.ts:77]:
```suggestion
 * @csspart textarea - Used to style the native textarea.
```

File: packages/fiori/src/Page.ts [hunk 8]
```diff
@@ -36,26 +36,15 @@ import PageCss from "./generated/themes/Page.css.js";
  * that there is enough space for the <code>ui5-page</code> to be rendered.
  * <b>Note:</b> In order for the <code>ui5-page</code> to be displayed, the parent element should have fixed height.
```
Comment (nnaydenow) [inline packages/fiori/src/Page.ts:37]: Let's add one <br /> tag here just to make the documentation to looks better.

**Review by nnaydenow — COMMENTED** [review]
At all it looks good to me. Just restore the unneeded files and address UA review.

File: packages/main/src/Slider.ts (hunk 5 above)
Comment (nikoletavnv) [reply packages/main/src/Slider.ts]: Done

File: packages/main/src/Slider.ts (hunk 6 above)
Comment (nikoletavnv) [reply packages/main/src/Slider.ts]: Done

File: packages/fiori/src/Page.ts (hunk 8 above)
Comment (nikoletavnv) [reply packages/fiori/src/Page.ts:38]: Done

File: packages/main/src/RangeSlider.ts (hunk 2 above)
Comment (nikoletavnv) [reply packages/main/src/RangeSlider.ts]: Done

File: packages/main/src/TextArea.ts (hunk 7 above)
Comment (nikoletavnv) [reply packages/main/src/TextArea.ts]: Done

File: packages/main/src/Slider.ts (hunk 4 above)
Comment (nikoletavnv) [reply packages/main/src/Slider.ts]: Done

**Pull request closed without merging**